Components/GraphTitle/index.js

```diff
@@ -0,0 +1,54 @@
+import Heading from "../Heading";
+import Paragraph from "../Paragraph";
+import { Container } from "../Container";
+import InputTypeText from "../InputTypeText";
+import { StyledInputTypeSubmit } from "../StyledInputTypeSubmit";
+
+export default function GraphTitle({
+  xLabel,
+  yLabel,
+  titleLabel,
+  onTitleChange,
+}) {
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  function handleTitle() {
+    if (titleLabel.length > 0) {
+      alert("Title of the graph is assigned");
+    }
+  }
+
+  function completedTitle() {
+    return titleLabel.length === 0;
+  }
```

Comment:
you can handle this directly in the argument of the InputTypeSubmit :)

Reply:
We wanted to be consistent with the other components. So we keep it like this.